### DISCUSSION TITLE
Work this also with HA and ValetudoRE Firmware on th vacuum ?

### DISCUSSION THREAD

**Discussion by MrPusemukel** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#488] [General]
Hi there , 
I had a question. Do this add on work with ValetudoRE Firmware on th vaccum also? 
I ask becouse i get an error "Failed Login" in the Camera card in my HA Dashboard
and i do not see any error in my configuration in Homeassistent.
Thanks for answer

**Comment by PiotrMachowski** [comment]
It doesn't, it connects to the Xiaomi cloud. Use this one instead:
https://github.com/sca075/valetudo_vacuum_camera/
